### ISSUE TITLE
Support for Rollup =< 1.0.0

### ISSUE THREAD

**Issue by ukjinjang** [jkuri/rollup-plugin-progress#11]
There is a warning message, if you use this plugin with Rollup =< v1.0.0.
```
The ongenerate hook used by plugin progress is deprecated. The generateBundle hook should be used instead.
```
This caused because, Rollup released v1.0.0 and some of their plugin hooks have become deprecated as you can see in [change log](https://github.com/rollup/rollup/blob/master/CHANGELOG.md#breaking-changes). The `ongenerate` hook deprecated and need to be replaced with `generateBundle`.

**Comment by jkuri** [comment]
feel free to open a pull request.

**Comment by ukjinjang** [comment]
@jkuri I made PR and please check it :)

**Comment by jkuri** [comment]
thank you @doublestat. I released this on npm as version 1.0.0.
